ambry-api/src/main/java/com/github/ambry/config/MySqlNamedBlobDbConfig.java

```diff
@@ -56,7 +56,6 @@ public class MySqlNamedBlobDbConfig {
    * A flag on whether to turn on DB Transition run
    */
   @Config(DB_TRANSITION)
-  @Default("false")
```

Comment:
revert

Reply:
Those Default are not used at all. It's ok to clean up. See
```
  public MySqlNamedBlobDbConfig(VerifiableProperties verifiableProperties) {
    this.dbInfo = verifiableProperties.getString(DB_INFO);
    this.localPoolSize = verifiableProperties.getIntInRange(LOCAL_POOL_SIZE, 5, 1, Integer.MAX_VALUE);
    this.remotePoolSize = verifiableProperties.getIntInRange(REMOTE_POOL_SIZE, 1, 1, Integer.MAX_VALUE);
    this.listMaxResults = verifiableProperties.getIntInRange(LIST_MAX_RESULTS, 100, 1, Integer.MAX_VALUE);
    this.dbTransition = verifiableProperties.getBoolean(DB_TRANSITION, false);
    this.dbRelyOnNewTable = verifiableProperties.getBoolean(DB_RELY_ON_NEW_TABLE, false);
  }
```